dnstap.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/dnstap/golang-dnstap"
+	"github.com/farsightsec/golang-framestream"
+	"github.com/golang/protobuf/proto"
+	"github.com/miekg/dns"
+	"github.com/patrickmn/go-cache"
+)
+
+const defaultTimeout = time.Hour
+
+var c = cache.New(defaultTimeout, defaultTimeout*2)
```

Comment:
Should use values derived from the config with sane defaults. The cache lib we use would be more beneficial if we could control the size of the cache and the expiry.